packages/@webex/internal-plugin-encryption/src/encryption.js

```diff
@@ -188,7 +188,9 @@ const Encryption = WebexPlugin.extend({
       return Promise.reject(new Error('Cannot encrypt `scr` without first setting `loc`'));
     }
 
-    return this.getKey(key, options).then((k) => scr.toJWE(k.jwk));
+    return this.getKey(key, options).then((k) =>
+      SCR.fromJSON(scr).then((encScr) => encScr.toJWE(k.jwk))
```

Comment:
can you add some comments on what this line does and why do we need to always do fromJSON arent there cases where its not in that case what would the function return , i assume old value